<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProcessDialog</class>
 <widget class="QDialog" name="CreateProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление нового процесса</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelPPID">
       <property name="text">
        <string>PPID</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditBasePriority">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditWorkTime">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelPID">
       <property name="text">
        <string>PID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditPID">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPPID">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditPriority">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPriority">
       <property name="text">
        <string>Приоритет</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelBasePriority">
       <property name="text">
        <string>Базовый приоритет</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelWorkTime">
       <property name="text">
        <string>Предполагаемое время выполнения</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditPID</tabstop>
  <tabstop>lineEditPPID</tabstop>
  <tabstop>lineEditPriority</tabstop>
  <tabstop>lineEditBasePriority</tabstop>
  <tabstop>lineEditWorkTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
